system/micd.py

```diff
@@ -2,15 +2,14 @@
 import numpy as np
 
 from cereal import messaging
-from openpilot.common.realtime import Ratekeeper
 from openpilot.common.retry import retry
 from openpilot.common.swaglog import cloudlog
+import threading
 
-RATE = 10
-FFT_SAMPLES = 4096
+FFT_SAMPLES = 1280
 REFERENCE_SPL = 2e-5  # newtons/m^2
-SAMPLE_RATE = 44100
-SAMPLE_BUFFER = 4096 # (approx 100ms)
+SAMPLE_RATE = 16000
```

Comment:
this reduces the frequency response so soundPressure is lower now. I tested that soundd still adjusts the volume for normal road noise and music.